Competition2018/src/main/java/competition/subsystems/gripperintake/GripperIntakeSubsystem.java

```diff
@@ -0,0 +1,64 @@
+package competition.subsystems.gripperintake;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.CommonLibFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+@Singleton
+public class GripperIntakeSubsystem extends BaseSubsystem {
+	
+	CommonLibFactory clf;
+	public XCANTalon rightMotor;
+	public XCANTalon leftMotor;
+	
+	
+	final DoubleProperty highPower;
+	final DoubleProperty lowPower;
+	
+	@Inject
+	public GripperIntakeSubsystem(CommonLibFactory clf, XPropertyManager propMan) {
+		this.clf = clf;
+		highPower = propMan.createPersistentProperty("Gripper Intake High Power", 1);
+		lowPower = propMan.createPersistentProperty("Gripper Intake Low Power", 0.25);
+	}
+	
+	public void temporaryHack() {
+		rightMotor = clf.createCANTalon(40);
+		leftMotor = clf.createCANTalon(40);
+	}
+	
+	/**
+	 * Directly controls motor power
+	 * @param power -1 intakes, +1 ejects
+	 */
```

Comment:
This comment isn't attached to anything. Did you used to have a `setPower(double power)` method here?